mne/epochs.py

```diff
@@ -1826,8 +1827,12 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
         if data.shape[0] != len(events):
             raise ValueError('The number of epochs and the number of events'
                              'must match')
-        tmax = (data.shape[2] - 1) / info['sfreq'] + tmin
-        if event_id is None:  # convert to int to make typing-checks happy
+        index_offset = 1 if data.shape[2] > 1 else 0
+        tmax = (data.shape[2] - index_offset) / info['sfreq'] + tmin
+        if index_offset == 0:
+            reject_tmin, reject_tmax = tmin, tmax
+        if event_id is None or isinstance(event_id, int):
```

Comment:
If I input a single event_id, this will make it so that `epochs.event_id` has multiple entries. That doesn't seem right?